data/repo_meta/rust.txt

```diff
@@ -84,3 +84,387 @@ tikv/tikv
 Keats/tera
 ibraheemdev/matchit
 smoltcp-rs/smoltcp
+oxc-project/oxc
```

Comment:
Do these new rust repos contain `\fuzz`?

Reply:
Yes they should